### PR TITLE
Switch `Header` component to v3 design tokens #DS-1453

### PR DESCRIPTION
**Pull request by adamkudrna**
### Additional context

image

### Issue reference

https://jira.almacareer.tech/browse/DS-1453

### CODE REVIEW

**Comment by literat** [comment]
In [Storybook](https://deploy-preview-1655--spirit-design-system-storybook.netlify.app/iframe.html?globals=&args=&id=components-header--playground&viewMode=story) there is a little glitch in the hamburger menu. Right after the click on the hamburger, it jumps a little bit to right. It can be noticed before the Offcanvas opens.

**Review by literat — APPROVED** [review]
LGTM 👍

File: packages/web/src/scss/components/Header/index.html [hunk 1]
```diff
@@ -2,17 +2,65 @@
...
-          width="65"
-          height="24"
-          alt="Spirit"
-        />
+    <header class="Header">
+      <a href="#" aria-label="JobBoard homepage">
+        <div class="ProductLogo">
+          <svg width="178" height="44" viewBox="0 0 178 44" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (crishpeen) [inline packages/web/src/scss/components/Header/index.html:11]: maybe we can either use `{{setVar "logo" "<svg...` to store the logo or add it to global Vite config runtimeOptions data and print it from there?

What do you think?

File: packages/web-twig/src/Resources/components/Header/stories/HeaderMinimal.twig [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<Header>
+    {% include '@components/Header/stories/_partials/JobBoardLogo.twig' %}
```
Comment (crishpeen) [inline packages/web-twig/src/Resources/components/Header/stories/HeaderMinimal.twig:2]: The implicit task is to implement ProductLogo in Twig? :D

**Review by crishpeen — APPROVED** [review]
We should imho discover way to make the logo in Web a partial. Otherwise LGTM

File: packages/web-twig/src/Resources/components/Header/stories/HeaderMinimal.twig (hunk 2 above)
Comment (adamkudrna) [reply packages/web-twig/src/Resources/components/Header/stories/HeaderMinimal.twig:2]: No, we no longer create new components in Twig 🙂.

File: packages/web-twig/src/Resources/components/Header/README.md [hunk 3]
```diff
@@ -107,10 +107,10 @@ just branding.
 
 ## API
 
-| Name       | Type                          | Default       | Required | Description                         |
-| ---------- | ----------------------------- | ------------- | -------- | ----------------------------------- |
-| `color`    | [`transparent` \| `inverted`] | `transparent` | ✕        | Color variant                       |
-| `isSimple` | `bool`                        | `false`       | ✕        | Shorter, centered version of Header |
+| Name       | Type                          | Default   | Required | Description                         |
+| ---------- | ----------------------------- | --------- | -------- | ----------------------------------- |
+| `color`    | \[`primary` \| `transparent`] | `primary` | ✕        | Color variant                       |
```
[the remark this answers is not in the document]
Comment (adamkudrna) [reply packages/web-twig/src/Resources/components/Header/README.md:112]: Yes.

**Pull request merged**